metrics/CSAF/DocumentCSAFFilenameValid/DocumentCSAFFilenameValid.yml

```diff
@@ -0,0 +1,11 @@
+# ====== Metadata ======
+id: DocumentCSAFFilenameValid
+category: CSAF
+description: This rule assesses whether a [SecurityAdvisoryDocument] that offers the property [RemoteDocumentLocation] has a valid filename in the property [path].
```

Comment:
To be consistent, let's add the "p1" tag to the property as well

Reply:
Do we want to delete the configuration part in that case, because we compare nothing and the target_value is not used.